bpdm-pool-api/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/api/PoolBpnApi.kt

```diff
@@ -62,4 +63,17 @@ interface PoolBpnApi {
     @Tag(name = ApiCommons.BPN_NAME, description = ApiCommons.BPN_DESCRIPTION)
     @PostMapping(CommonApiPathNames.SUBPATH_SEARCH)
     fun findBpnsByIdentifiers(@RequestBody request: IdentifiersSearchRequest): ResponseEntity<Set<BpnIdentifierMappingDto>>
+
+    @Operation(
+        summary = "Return BPNL/S/A based on the requested identifiers",
+        description = "Find business partner numbers by requested-identifiers."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Found bpn to based on the requested identifiers"),
+        ]
+    )
+    @Tag(name = ApiCommons.BPN_NAME, description = ApiCommons.BPN_DESCRIPTION)
+    @PostMapping("/request-ids")
+    fun findBpnByRequestedIdentifiers(@RequestBody request: Set<String>): ResponseEntity<Set<BpnRequestIdentifierMappingDto>>
```

Comment:
For new endpoints we prefer to use wrapper objects for the request body instead of naked Json arrays. Please create a request object containing the list of request identifiers. This increases backwards compatible extensibility,   gives us the opportunity to add more documentation and makes it easier for the user to understand what they should pass in the request body. 

As examples you can have a look at the legal-entites/search or /members/legal-entities/search endpoints.

Reply:
Use the wrapper classes for the new POST endpoint.